quantumdb-core/src/main/java/io/quantumdb/core/schema/definitions/Table.java

```diff
@@ -116,7 +116,7 @@ public boolean containsIndex(Collection<String> columns) {
 		checkArgument(!columns.isEmpty(), "You must specify at least one entry in 'columns'.");
 
 		return indexes.stream()
-				.anyMatch(c -> c.getColumns().equals(Lists.newArrayList(columns)));
+				.anyMatch(c -> Sets.newHashSet(c.getColumns()).equals(Sets.newHashSet(columns)));
```

Comment:
Order of columns might change when a column name has been changed.

Reply:
Yup, fair enough.